moreutils/__init__.py

```diff
@@ -17,8 +17,8 @@ async def setup_after_ready(bot):
             for alias in command.aliases:
                 if bot.get_command(alias):
                     command.aliases[command.aliases.index(alias)] = f"mu{alias}"
-    bot.add_cog(cog)
+    await bot.add_cog(cog)
 
 
-def setup(bot):
-    create_task(setup_after_ready(bot))
+async def setup(bot):
+    await create_task(setup_after_ready(bot))
```

Comment:
[create_task](https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task) is sync function
```suggestion
    create_task(setup_after_ready(bot))
```